content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+
+To update an existing Bevy App or Plugin to **Bevy 0.6**, check out our [0.5 to 0.6 Migration Guide](/learn/book/migration-guides/0.5-0.6/).
+
+There are a _ton_ of improvements in this release, but here are some of the big ones: 
+
+* A brand new modern renderer that is prettier, faster, and easier to extend
+* Directional and point light shadows
+* Clustered forward rendering
+* Frustum culling
+* Significantly faster sprite rendering with less boilerplate
+* Native WebGL2 support
+* High level custom Materials
+* More powerful shaders: preprocessors, imports, WGSL support
+* Bevy ECS ergonomics and performance improvements. No more `.system()`!
+
+Read on for details!
+
+<!-- more -->
+
+## The New Bevy Renderer
+
+**Bevy 0.6** introduces a brand new modern renderer that is:
+* **Faster**: More parallel, less computation per-entity, more efficient CPU->GPU dataflow, and (soon-to-be-enabled) pipelined rendering
+* **Simpler**: Fewer layers of abstraction, simpler data flow, improved low-level, mid-level, and high-level interfaces, direct wgpu access 
+* **Modular to its core**: Standardized 2d and 3d core pipelines, extensible Render Phases and Views, composable entity/component-driven draw functions, shader imports, extensible and repeatable render pipelines via "sub graphs"
+* **Prettier**: We're releasing the new renderer alongside a number of graphical improvements, such as directional and point light shadows, clustered forward rendering (so you can draw more lights in a scene), and spherical area lights. We also have a ton of new features in development (cascaded shadow maps, bloom, particles, shadow filters, and more!)
+* **Industry Proven**: It takes inspiration from battle tested renderer architectures, such as [Bungie's pipelined Destiny renderer](https://advances.realtimerendering.com/destiny/gdc_2015/Tatarchuk_GDC_2015__Destiny_Renderer_web.pdf). We also learned a lot from (and worked closely with) other renderer developers in the Rust space, namely @aclysma ([rafx](https://github.com/aclysma/rafx)) and @cwfitzgerald ([rend3](https://github.com/BVE-Reborn/rend3)). The New Bevy Renderer wouldn't be what it is without them and I highly recommend checking out their projects!
+
+I promise I'll qualify all of those fluffy buzz words below. I am confident that the New Bevy Renderer will be a rallying point for the Bevy graphics ecosystem and (hopefully) the Rust graphics ecosystem at large. We still have _plenty_ of work to do, but I'm proud of what we have accomplished so far and I'm excited for the future!
```

Comment:
```suggestion
I promise I'll qualify all of those fluffy buzzwords below. We still have _plenty_ of work to do, but I'm proud of what we've built so far and excited for the future!
```

"buzzword" is one word. Editing for brevity.

Reply:
I felt the removed sentence was important and valuable as it highlights community value and would prefer not to remove it. :) I have a strong feeling of collaboration around rendering things with various people in the Rust space and that has been cultivated or has come to the fore perhaps primarily during this release.

Reply:
Yeah I think I'll just throw in the buzzword fix. I like the current phrasing.